### PR TITLE
added main.js and style.css

### PR DESCRIPTION
**Pull request by Pitclair**
functions in main.js have doc comments
style.css is mandatory for main.js functions to work
a list of necessary requirements for running the js function is added to the main.js

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+
+inside body:
+
+    <div id="mapid"></div>
+
+
+    <script src="main.js"></script>
+	<script src="https://code.jquery.com/jquery-3.4.1.min.js"
+	    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```
Comment (farzad1132) [inline main.js]: these includes exist in the map.html

File: main.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+    maxZoom: 6,
+    id: 'mapbox/streets-v11',
+    // x: 53,
+    // y: 33,
+    // z: 1,
+    tileSize: 512,
+    zoomOffset: -1,
+    accessToken: 'pk.eyJ1IjoicGl0Y2xhaXIiLCJhIjoiY2s3MDZmZGcwMDB1NDNtcDBmZ3pjdWp2NyJ9.MC35oYwFqBb-Zndvv2yiHQ'
+}).addTo(mymap);
```
Comment (farzad1132) [inline main.js]: also these lines

**Comment by farzad1132** [comment]
also, point where I have to pass the link click event?

**Comment by Pitclair** [comment]
updated version - with only the main functions
[updated-js-and-css.zip](https://github.com/farzad1132/NetPlanner/files/4333303/updated-js-and-css.zip)

**Pull request merged**